### PR TITLE
Create Makefile

### PR DESCRIPTION
**Pull request by naresh-kumar-babu**
Resolving problems in #53

### CODE REVIEW

**Review by yaronkoren — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+APP_NAME="canasta"
+
+build:
```
Comment (bawolff) [inline Makefile]: extra space after colon

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		@echo "To run the CLI you have just generated, please call it as \033[0;32m./${APP_NAME}\033[0m instead of ${APP_NAME}."
+
+prepare-lint:
+		@if ! hash golangci-lint 2>/dev/null; then printf "\e[1;36m>> Installing golangci-lint (this may take a while)...\e[0m\n"; go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest; fi
+
+lint:
+		@printf "\e[1;36m>> golangci-lint\e[0m\n"
+		@golangci-lint run ./...
+
+help:
```
Comment (bawolff) [inline Makefile]: extra space after colon.

**Review by bawolff — COMMENTED** [review]
So overall looks good.

This doesn't set the same LDFLAGS as build.sh does. I do not know if that is important or not.

**Comment by bawolff** [comment]
[Pointing out the extra space is just because it looks like an accident. If this is part of some coding convention thing, please ignore the nitpick]

**Review by bawolff — APPROVED** [review]

**Pull request merged**